pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -140,8 +140,26 @@ let
       ./patches/fix-freetype.patch
       ./patches/nix_plugin_paths_68.patch
       ./patches/remove-webp-include-69.patch
-    ] ++ optional enableWideVine ./patches/widevine.patch;
-
+    ] ++ optional enableWideVine ./patches/widevine.patch
+      ++ optional ((versionRange "69" "70") && stdenv.isAarch64)
+           (fetchpatch {
+              url    = https://raw.githubusercontent.com/OSSystems/meta-browser/master/recipes-browser/chromium/files/0001-vpx_sum_squares_2d_i16_neon-Make-s2-a-uint64x1_t.patch;
```

Comment:
(Applies to all the patches) These patch urls are from the repo's  `master` branch, content may change and result in hash mismatch.

It would be better to pin them to a suitable git revision, like https://raw.githubusercontent.com/OSSystems/meta-browser/e4a667deaaf9a26a3a1aeb355770d1f29da549ad/recipes-browser/chromium/files/0001-vpx_sum_squares_2d_i16_neon-Make-s2-a-uint64x1_t.patch